onnxruntime/core/providers/webnn/builders/impl/activation_op_builder.cc

```diff
@@ -66,13 +72,35 @@ Status ActivationOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder,
   return Status::OK();
 }
 
+// Operator support related.
+bool ActivationOpBuilder::IsOpSupportedImpl(const InitializedTensorSet& /* initializers */,
+                                            const Node& node,
+                                            WebnnDeviceType /* device_type */,
+                                            const logging::Logger& logger) const {
+  const auto& input_defs = node.InputDefs();
+
+  std::vector<int64_t> input_shape;
+  if (!GetShape(*input_defs[0], input_shape, logger))
+    return false;
+
+  if (node.OpType() == "Gelu") {
+    NodeAttrHelper helper(node);
+    if (helper.Get("approximate", "none") != "none") {
```

Comment:
We should just return true in this case. I would be more worried about the *reverse* case (if we supported approximate `tanh` but didn't support `erf`), but if `erf` is more accurate than `approximate`, then it yields no benefit to precision to fall back to the CPU, and it hurts perf badly to split partitions.